hazelcast/src/main/java/com/hazelcast/collection/impl/queue/QueueItem.java

```diff
@@ -96,10 +96,10 @@ public void readData(ObjectDataInput in) throws IOException {
     }
 
     @Override
-    public int compareTo(QueueItem o) {
-        if (itemId < o.getItemId()) {
+    public int compareTo(QueueItem other) {
+        if (getItemId() < other.getItemId()) {
```

Comment:
What is the advantage of using a getter here?

Reply:
Nope. Will revert back.